<?xml version="1.0"?>
<launch>

    <include file="$(find rbc_core)/launch/machines.launch" />

    <!-- ros_control server, provide ControllerManager which allows controller 
    plugins (in this case our custom OmniwheelBaseController) to access them -->
    <node pkg="rbc_base" type="robot_hw" name="robot_hw" machine="rbc_bot" output="screen"/>

    <!-- rosparam configuration required to tell ros_control that OmniwheelBaseController is the controller we will use -->
    <rosparam file="$(find rbc_base)/config/controller.yaml" command="load"/>

    <!-- Load the controller -->
    <node pkg="controller_manager" type="spawner" name="controller_spawner" machine="rbc_bot" output="screen"
        args="rbc_base/OmniwheelBaseController joint_state_controller/JointStateController"/>

</launch>